src/AutoRest.CSharp/Common/Output/Models/MethodSignatureBase.cs

```diff
@@ -4,10 +4,15 @@
 using System;
 using System.Collections.Generic;
 using AutoRest.CSharp.Output.Models.Shared;
+using AutoRest.CSharp.Utilities;
 
 namespace AutoRest.CSharp.Output.Models
 {
-    internal abstract record MethodSignatureBase(string Name, string? Description, MethodSignatureModifiers Modifiers, IReadOnlyList<Parameter> Parameters);
+    internal abstract record MethodSignatureBase(string Name, string? Description, string? Summary, MethodSignatureModifiers Modifiers, IReadOnlyList<Parameter> Parameters)
+    {
+        public string? SummaryText => Summary.IsNullOrEmpty() ? Description : Summary;
+        public string? DescriptionText => Summary.IsNullOrEmpty() ? string.Empty : Description;
```

Comment:
I remember the decision is that we just follow the M4 output? So if M4 copies the `Summary` into `Description`, we will have duplicated texts. If it's empty, then we just don't print it.

Reply:
I didn't get. I do follow this logic. If there is duplicate text, I will put them both into summary and description.